apps/web/src/design-system/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { TextInput } from './TextInput';
+
+export default {
+  title: 'Input/TextInput',
+  component: TextInput,
+  argTypes: {
+    value: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} as ComponentMeta<typeof TextInput>;
+
+const Template: ComponentStory<typeof TextInput> = ({ ...args }) => <TextInput {...args} />;
+
+export const PrimaryUse = Template.bind({});
+PrimaryUse.args = {
+  label: 'Notification Name',
+  placeholder: 'Notification name goes here...',
+};
+
```

Comment:
Also missing the description prop here:
![CleanShot 2022-02-08 at 13 00 47@2x](https://user-images.githubusercontent.com/8872447/152974339-f17cf88c-d695-46a2-b562-127c4a2ac309.png)

Reply:
description added